<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreForm</class>
 <widget class="QWidget" name="ScoreForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成绩窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="student">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>成绩查询</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="cbxAll">
            <property name="text">
             <string>全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxAllMust">
            <property name="text">
             <string>公必</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxAllChoose">
            <property name="text">
             <string>公选</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxMyMust">
            <property name="text">
             <string>专必</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxMyChoose">
            <property name="text">
             <string>专选</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="myScoreTableView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>311</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTabWidget" name="studentTabWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>柱状图</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" colspan="2">
            <widget class="QChartView" name="studentBarChartView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>饼状图</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" colspan="2">
            <widget class="QChartView" name="studentPieChartView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="teacher">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>课程：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>打分列表</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="studentTableView">
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ptbSave">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeWidget" name="scoreSituationTree">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>212</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>分数段</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>人数</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <item>
           <property name="text">
            <string>60分以下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>70分以下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>80分以下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90分以下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90分及以上</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>最高分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>最低分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>平均分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>未打分</string>
           </property>
          </item>
         </widget>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>柱状图</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="pbtBarNew">
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>358</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QChartView" name="teacherBarView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>饼状图</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QPushButton" name="pbtPieNew">
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>358</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QChartView" name="teacherPieView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
